operator/builtin/input/tcp/tcp.go

```diff
@@ -32,9 +32,9 @@ import (
 )
 
 const (
-	// minBufferSize is the initial size used for buffering
+	// minLogSize is the initial size used for buffering
 	// TCP input
-	minBufferSize = 64 * 1024
+	minLogSize = 64 * 1024
```

Comment:
Nit: I know I just suggested this name, but it's actually quite difficult to reason about its actual purpose here. 

The value actually represents "the minimum allowable maximum log size", so maybe `minMaxLogSize`? This is not great either, but at least makes sense next to `DefaultMaxLogSize`. 

I'm open to suggestions here, or we can just leave it (as long as the comment is clearer)

Reply:
naming is not my best skill, but to keep convention I changed it to `minMaxLogSize`

Reply:
This is a weird one to try to name. Thanks for humoring my nit picking.